src/lib.rs

```diff
@@ -22,6 +22,8 @@ mod fft;
 pub mod field;
 pub mod flp;
 mod fp;
+#[cfg(feature = "experimental")]
+pub mod idpf;
```

Comment:
Clients of crate `prio` should be using the eventual `Poplar1` VDAF, not the underlying `idpf` module. Can this be `pub(crate)` or otherwise restricted?

Reply:
I think it's useful to export. For example, I might end up implementing (a prototype of) Doplar elsewhere. However since it's behind `experimental` it won't show up with the normal feature set anyway.

Incidentally: What's our goal for API breaking changes for `experimental`? Do we care if we make breaking changes within the same version?

Reply:
We will also need to either export or wrap `IdpfInput`, for use in the eventual Poplar1 API.

Reply:
> Incidentally: What's our goal for API breaking changes for `experimental`? Do we care if we make breaking changes within the same version?

IMO anything experimental is unstable, i.e. the API can break without a major version change.

> We will also need to either export or wrap `IdpfInput`, for use in the eventual Poplar1 API.

Oh, OK, that's good enough for me. Just wanted to make sure we're deliberately committing to public API surface.